.github/workflows/smoke-test.yml

```diff
@@ -120,26 +120,39 @@ jobs:
           sudo docker run --restart=always -d jenkins/jenkins
           sudo docker run --restart=always -d nginx
           sudo docker run --restart=always -d -e POSTGRES_HOST_AUTH_METHOD=trust postgres
+          # NOTE: adjust this accordingly with the commands above if needed
+          UP_CONTAINERS_OLD=6
 
           # Wait for the restart policy to take effect
           # See: https://docs.docker.com/engine/containers/start-containers-automatically/#restart-policy-details
           sleep 10
```

Comment:
Add
```
echo "List of running containers"
docker ps -a
```
and possibly a check to know if a failure was caused by normal container startup error or the subsequent refresh.

Reply:
The trap just prints logs in case of a failure. Nothing is checked there.